src/main/java/ch/njol/skript/conditions/CondIsFuel.java

```diff
@@ -0,0 +1,60 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.Material;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+
+@Name("Is Fuel")
+@Description("Checks whether an item can be used as fuel in a furnace.")
+@Examples({"on right click on furnace:",
+	"\tif player's tool is not fuel:",
+	"\t\tsend \"Please hold a valid fuel item in your hand\"",
+	"\t\tcancel event"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("1.11+")
```

Comment:
Looks like this was added in 1.11.2
Should be changed just for extra clarity (even though Skript doesn't officially any other 1.11.x version)
```suggestion
@RequiredPlugins("1.11.2+")
```